### ISSUE TITLE
Typescript typings: Could not find a declaration file for module 'ol/style/Style'.

### ISSUE THREAD

**Issue by jt-helsinki** [openlayers/openlayers#9170]
OpenLayers ver. 5.3.0

Using typescript, I am trying to use the included type definitions but I am running into a number of problems with trying to use the included type definitions. Although it appears to be OK I think this problem might have something to do with how the `tsconfig.json` is set up. However this is a best guess. 

In the `tsconfig.json` below I am trying to scan the `ol` package under the `node_modules` directory for the `ol` types. 

*Note:* I am not using the `@types/ol` typings. I am using the typings included with openlayers.

**To Reproduce**

**tsconfig.json**
```
{
  "compileOnSave": true,
  "compilerOptions": {
    "baseUrl": ".",
    "outDir": "build/dist",
    "module": "esnext",
    "target": "es5",
    "lib": [
      "es5",
      "es6",
      "dom"
    ],
    "sourceMap": true,
    "allowJs": true,
    "checkJs": true,
    "jsx": "react",
    "moduleResolution": "node",
    "rootDir": "src",
    "forceConsistentCasingInFileNames": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noImplicitAny": true,
    "strictNullChecks": false,
    "suppressImplicitAnyIndexErrors": true,
    "noUnusedLocals": false,
    "noUnusedParameters": false,
    "pretty": true
  },
  "include": [
    "src/*",
    "node_modules/ol/**/*.js"
  ],
  "exclude": [
    "node_modules",
    "build",
    "scripts",
    "acceptance-tests",
    "webpack",
    "jest",
    "src/setupTests.ts"
  ]
}
```
**Error message:**

> (1,26): Could not find a declaration file for module 'ol/coordinate'. '/Users/xyz/Documents/workspace/app/node_modules/ol/coordinate.js' implicitly has an 'any' type.
  Try `npm install @types/ol` if it exists or add a new declaration (.d.ts) file containing `declare module 'ol/coordinate';`

**Expected behavior**
The code should compile as intended and all `ol` types should be recognised by typescript compiler and the IDE (Intellij).

**Comment by ahocevar** [comment]
OpenLayers does not have included types. I think you are using the [@types/ol](https://npmjs.com/package/@types/ol) package from Definitely Typed. If so, you should create a bug report there.

**Comment by ahocevar** [comment]
We are aware of these issues and trying to fix them for the 6.0 release.
